docs/en/integrations/data-ingestion/apache-spark/spark-native-connector.md

```diff
@@ -0,0 +1,583 @@
+| `Map`                                                             | `MapType`                      | ✅         | No           | Keys are limited to `StringType`                   |
+| `IntervalYear`                                                    | `YearMonthIntervalType(Year)`  | ✅         | Yes          |                                                    |
+| `IntervalMonth`                                                   | `YearMonthIntervalType(Month)` | ✅         | Yes          |                                                    |
+| `IntervalDay`, `IntervalHour`, `IntervalMinute`, `IntervalSecond` | `DayTimeIntervalType`          | ✅         | No           | Specific interval type is used                     |
+| `Object`                                                          |                                | ❌         |              |                                                    |
+| `Nested`                                                          |                                | ❌         |              |                                                    |
+| `Tuple`                                                           |                                | ❌         |              |                                                    |
+| `Point`                                                           |                                | ❌         |              |                                                    |
+| `Polygon`                                                         |                                | ❌         |              |                                                    |
+| `MultiPolygon`                                                    |                                | ❌         |              |                                                    |
+| `Ring`                                                            |                                | ❌         |              |                                                    |
+| `IntervalQuarter`                                                 |                                | ❌         |              |                                                    |
+| `IntervalWeek`                                                    |                                | ❌         |              |                                                    |
+| `Decimal256`                                                      |                                | ❌         |              |                                                    |
+| `AggregateFunction`                                               |                                | ❌         |              |                                                    |
+| `SimpleAggregateFunction`                                         |                                | ❌         |              |                                                    |
+
+### Inserting data from Spark into ClickHouse
```

Comment:
We use write in the entire doc i think we should use it here

Reply:
`write` data is the API name in Spark, the headline is meant to be indicative for users.